src/docs/sdk/features.mdx

```diff
@@ -231,6 +231,58 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 - span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
 - when network error occurs, span status must be set to `internal_error`
 
+### HTTP Client errors
+
+HTTP Client errors are automatically captured and sent to `https://sentry.io`.
+
+The HTTP Client integration should have 3 configuration options:
+
+* `captureFailedRequests` defaults to `false` due to PII reasons.
+* `failedRequestStatusCode` defaults to `500` to `599`, this configuration option accepts a `List` of `HttpStatusCodeRange` which is a range of HTTP status code -> `min` to `max` or a single `status_code`.
+* `failedRequestsTargets` defaults to `.*`, this configuration option accepts a `List` of `String` that may be Regular expressions as well, similar to <Link to="/sdk/performance/#tracepropagationtargets">tracePropagationTargets</Link>.
+
+Then the HTTP Client Integration should not capture the error:
+
+* SDKs won't capture error events if `captureFailedRequests` is disabled.
+* SDKs won't capture error events if the HTTP Request url isn't a match for any of the `failedRequestsTargets`.
+* SDKs won't capture error events if the HTTP Response status code isn't within the defined ranges in `failedRequestStatusCode`.
+
+The SDKs should capture error events with the following properties:
+
+The Request interface, See the <Link to="/sdk/event-payloads/request/">Spec</Link> for details.
+
+The Response context (Not yet under the Spec):
```

Comment:
Linking to the Spec when Relay is done, See [RFC](https://github.com/getsentry/rfcs/blob/main/text/0022-response-context.md).

Reply:
I believe the response content would also be something nice to have.

Reply:
This is the response content.
If you mean the request, it's linked in the original spec (line above), where there is the JSON content, I don't want to duplicate content, one more thing to maintain and to keep up to date.
This JSON example is going to be removed once this comment https://github.com/getsentry/develop/pull/711/files#r994681788 is addressed but it's not dependent on me.